.version

```diff
@@ -1 +1 @@
-1.6.6
+1.7.0
```

Comment:
This change will not be backwards compatible, so the minor version is bumped to infer it's receiving _new_ content to an important component.